components/hostobjects/org.jaggeryjs.hostobjects.xhr/src/main/java/org/jaggeryjs/hostobjects/xhr/XMLHttpRequestHostObject.java

```diff
@@ -594,12 +597,18 @@ private static boolean isInvalidHeader(String header) {
         return false;
     }
 
-    private ProxyHost getProxyConfig() {
+    private static ProxyHost getProxyConfig(String host) {
 
         ProxyHost proxyConfig = null;
 
-        String proxyHost = System.getProperty("http.proxyHost");
-        String proxyPortStr = System.getProperty("http.proxyPort");
+        String proxyHost = System.getProperty(HTTP_PROXY_HOST);
+        String proxyPortStr = System.getProperty(HTTP_PROXY_PORT);
+        String nonProxyHosts = System.getProperty(HTTP_NON_PROXY_HOSTS);
+
+
```

Comment:
remove the unnecessary line

Reply:
Fixed in af3ac874ea63a07e139fdd85b36eeeed3d15e410